src/main/java/uk/gov/hmcts/reform/refunds/repository/RefundsRepository.java

```diff
@@ -42,11 +42,11 @@ Optional<List<Refund>> findByRefundStatusAndUpdatedByIsNotAndServiceTypeInIgnore
     Optional<List<Refund>> findByNotificationSentFlag(String notificationSentFlag);
 
     @Query("select rf from Refund rf "
-        + "where rf.dateUpdated NOT between ?1 and ?2 AND (rf.refundStatus = 'Approved' or rf.refundStatus = 'Accepted')")
+        + "where rf.dateUpdated NOT between ?1 and ?2 AND (rf.refundStatus.name = 'Approved' or rf.refundStatus.name = 'Accepted')")
     List<Refund> findByDatesBetween(Date fromDate, Date toDate);
 
     @Query("select rf from Refund rf "
-        + "where rf.paymentReference = ?1  AND (rf.refundStatus = 'Approved' or rf.refundStatus = 'Accepted')"
+        + "where rf.paymentReference = ?1  AND (rf.refundStatus.name = 'Approved' or rf.refundStatus.name = 'Accepted')"
```

Comment:
why is this changed now?

Reply:
Newer hibernate (>=6.2) is more strict on data types, and it couldn't compare Enum with string, therefore needed to change to compare string with string